doc/classes/AStar2D.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AStar2D" inherits="Reference" category="Core" version="3.2">
+	<brief_description>
+		AStar class representation that uses 2d-vectors as edges.
+	</brief_description>
+	<description>
+		This is a wrapper for [AStar] class which using 2d-vectors instead 3d.
```

Comment:
uses instead of using

Reply:
`Instead of` not just `instead`.
`This is a wrapper for the [AStar] class which uses 2D vectors instead of 3D vectors.`

Reply:
Fixed